help/en/html/hardware/cmri/CMRI.shtml

```diff
@@ -938,20 +944,21 @@ for easy access click an item in this list:</p>
 
   <p>The following screen shows polling with responses for nodes 20 and 21. </p>
 
-  <p><img alt="" src="images/cmrinetmonitor-pollrespwtext.png" width="858"
-  height="518"></p>
+  <p><img alt="" src="images/cmrinetmonitor-pollresponse.png" width="997"
+  height="480"></p>
 
   <p>There are display option checkboxes which provide additional information
   and formatting for the messages.</p>
 
-  <p>If the<strong>Show Timestamps</strong> check box option is checked, each
+  <p>If the <strong>Show Timestamps</strong> check box option is checked, each
   message is prefixed with the system time in the form of hh:mm:ss:mss. The
   message type (e.g Receive) and node address (ua) are displayed next. The "IB"
   are the input data bytes, displayed as hexadecimal digits, one digit per 8
   bit byte.</p>
-
+</ul>
```

Comment:
See comments above about allowed list contents.

Reply:
Corrected lines in error.